legacy/contracts/risk-management/AssetList.sol

```diff
@@ -3,68 +3,37 @@ pragma solidity ^0.4.21;
 import "../dependencies/Owned.sol";
 
 // Generic AssetList
+// should this inherit from Policy?
```

Comment:
don't think so since it does not implement `rule` or `position`; looks like it is just used as a list in a couple other rules